src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageReadmeControl.xaml.cs

```diff
@@ -43,7 +43,7 @@ public PackageReadmeControl()
         /// </summary>
         public bool IsBusy
         {
-            get => _isBusy || ReadmeViewModel.IsBusy;
+            get => _isBusy || (ReadmeViewModel?.IsBusy).GetValueOrDefault(false);
```

Comment:
```suggestion
            get => _isBusy || ReadmeViewModel?.IsBusy ?? false;
```

fewer characters, and maybe easier to understand?   Also results in [less IL](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBLANgHwAEAmARgFgAoQgBgAJDSA6ASQHkBuKwgZgeLoBhOgG8qdCXQAOUbADcAhhhh0AyhlgKAtnQDOGmNq6VJDPsAgRcdALIAKAJSjxpyYQDseg9oD8TQQoAdgBKhgAmdD4+dABmCri6MMamAL4uErx0Fla2xI7OJq4Znnb6mlp+ASHhDkwA4jAYAGrxAK4wbFAAIjBxrbgYdnEJMA7JkmmUKUA===) generated.

Reply:
So I applied it and then it got mad because can't do bool || bool?

Reply:
`ReadmeViewModel?.IsBusy == true`
?

Reply:
+1 to make the change. Whichever suggestion can get working is fine.